src/main/kotlin/io/github/gradlenexus/publishplugin/internal/NexusClient.kt

```diff
@@ -97,6 +97,28 @@ class NexusClient(private val baseUrl: URI, username: String?, password: String?
         }
     }
 
+    fun closeStagingRepository(stagingRepositoryId: String) {
+        try {
+            val response = api.closeStagingRepo(Dto(StagingRepositoryToTransit(listOf(stagingRepositoryId), "Closed by io.github.gradle-nexus.publish-plugin Gradle plugin"))).execute()
+            if (!response.isSuccessful) {
+                throw failure("close staging repository", response)
+            }
+        } catch (e: IOException) {
+            throw UncheckedIOException(e)
```

Comment:
Is it required?

Reply:
Hmm, it's most likely a leftover from the times the code was written in Java :).